POS/UserInterface/01c_CashRegister.md

```diff
@@ -3,7 +3,7 @@
 [!!Manage the cash register](../Operation/05_ManageCashRegister.md)
 
 # Cash register
-
+<!---Error-->
 *Venduo POS > Sales > Select Store and Pay Desk > Tab CASH REGISTER*
 
```

Comment:
- Cash in/out -> Schrägstrich ist auch "problematisch" in den Headers: die funktionieren in GitHub nicht, aber schon auf der Webseite. S. ESG: https://actindo.atlassian.net/wiki/spaces/DOC/pages/96796734/General#Links

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/a4f0ab52-a6b4-48cb-8866-6c60721c47ba)
 
 Alternativ "Cash in and Cash out" (in Header)?